obspy/signal/src/evalresp/resp_fctns.c

```diff
@@ -401,25 +403,57 @@ void check_channel(struct channel *chan) {
       if(ref_flag && deci_flag) {
         stage_ptr->first_blkt = ref_blkt;
         ref_blkt->next_blkt = filt_blkt;
+        if (filt_blkt == NULL) {
```

Comment:
... and added these checks to be sure there were no `NULL`-dereferences, but it seems to break `test_response_54_without_58`. It doesn't make sense to me that blockettes should be re-used for stages, but maybe I misunderstood something.